syserol/dataImport.py

```diff
@@ -15,6 +15,8 @@ def load_file(name):
 def importLuminex(antigen=None):
     """ Import the Luminex measurements. Subset if only a specific antigen is needed. """
     df = load_file("data-luminex")
+    # Delete antigens with more than 97% missingness
```

Comment:
This code doesn't select based on missingness levels, so this comment should be changed to reflect the code.